Bluetooth: Mesh: Light controller example (#2438)
nrfconnect/sdk-nrf: samples/bluetooth/mesh/light_ctrl/README.rst

   | Gen. OnOff Server             |                            |
   +-------------------------------+----------------------------+
   | Gen. DTT Server               |                            |
   +-------------------------------+----------------------------+
   | Gen. Power OnOff Server       |                            |
   +-------------------------------+----------------------------+
   | Gen. Power OnOff Setup Server |                            |
   +-------------------------------+----------------------------+
   | Light Lightness Server        |                            |
   +-------------------------------+----------------------------+
   | Light Lightness Setup Server  |                            |
   +-------------------------------+----------------------------+

The models are used for the following purposes:

- The first element consists of a Config Server and a Health Server.
  The Config Server allows configurator devices to configure the node remotely.
  The Health Server provides ``attention`` callbacks that are used during provisioning to call your attention to the device.
  These callbacks trigger blinking of the LEDs.

> remotely, and the Health Server provides ``attention`` callbacks that are used during provisioning to call your attention to
> the device. These callbacks trigger blinking of the LEDs. The eight other models contained within the first element is the result
> of a single implementation of the Light Lightness Server. These models are bound internally to the light source hardware you wish
> to control (in this case the first LED of the DK). External control of the light source can be achieved directly by using either
> the Gen. Level Server or the Gen. OnOff Server in the first element.
> 
> The three models contained within the second element is the result of a single implementation of the Light Control Server.
> Together they establish a controller that can, if enabled, take control over a Light Lightness Server instance, where the
> controller decides parameters such as fade time, lighting levels for different states, and inactivity timing. In this sample
> the Light Control Server is bound internally to the Light Lightness Server instance in the second element and enabled by default on boot up.
> External control of the light source associated with the Light Lightness Server can be achieved indirectly by the Light Control
> Server by using the Gen. OnOff Server in the second element.

Reviewer: ```suggestion
- The eight other models contained within the first element are a result of a single implementation of the Light Lightness Server.
  These models are bound internally to the light source hardware you want to control (in this case the first LED of the DK).
  External control of the light source can be achieved directly by using either the Generic Level Server or the Generic OnOff Server in the first element.
- The three models contained within the second element are a result of a single implementation of the Light Control Server.
  Together they establish a controller that, if enabled, can take control over a Light Lightness Server instance, where the controller decides parameters such as fade time, lighting levels for different states, and inactivity timing.
  In this sample the Light Control Server is bound internally to the Light Lightness Server instance in the second element, and enabled by default on boot up.
  External control of the light source associated with the Light Lightness Server can be achieved indirectly by the Light Control Server, by using the Generic OnOff Server in the second element.
```
Something happened here, it does not seem like the rest of the bullets were integrated. Try again :)

Author: Ah, sry... Must have happend during a rebase.
